src/examples/java/com/amazonaws/crypto/examples/keyring/rawaes/RawAes.java

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.amazonaws.crypto.examples.keyring.rawaes;
+
+import com.amazonaws.encryptionsdk.AwsCrypto;
+import com.amazonaws.encryptionsdk.AwsCryptoResult;
+import com.amazonaws.encryptionsdk.DecryptRequest;
+import com.amazonaws.encryptionsdk.EncryptRequest;
+import com.amazonaws.encryptionsdk.keyrings.Keyring;
+import com.amazonaws.encryptionsdk.keyrings.StandardKeyrings;
+
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.SecureRandom;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This examples shows how to configure and use a raw AES keyring.
+ * <p>
+ * https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/choose-keyring.html#use-raw-aes-keyring
+ * <p>
+ * In this example, we use the one-step encrypt and decrypt APIs.
+ */
+public class RawAes {
+
+    /**
+     * Demonstrate an encrypt/decrypt cycle using a raw AES keyring.
+     *
+     * @param sourcePlaintext Plaintext to encrypt
+     */
+    public static void run(final byte[] sourcePlaintext) {
+        // Instantiate the AWS Encryption SDK.
+        final AwsCrypto awsEncryptionSdk = new AwsCrypto();
+
+        // Prepare your encryption context.
+        // https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#encryption-context
+        final Map<String, String> encryptionContext = new HashMap<>();
+        encryptionContext.put("encryption", "context");
+        encryptionContext.put("is not", "secret");
+        encryptionContext.put("but adds", "useful metadata");
+        encryptionContext.put("that can help you", "be confident that");
+        encryptionContext.put("the data you are handling", "is what you think it is");
+
+        // Generate an AES key to use with your keyring.
+        //
+        // In practice, you should get this key from a secure key management system such as an HSM.
+        SecureRandom rnd = new SecureRandom();
+        byte[] rawKey = new byte[16]; // 128 bits
```

Comment:
We should show using AES-256.

```suggestion
        byte[] rawKey = new byte[32]; // 256 bits
```